src/locales/en-US.ts

```diff
@@ -1,4 +1,5 @@
 import { PluginFormEnUS } from '@/components/PluginForm';
+import { ActionBarEnUS } from '@/components/ActionBar';
```

Comment:
Don't need this line any more, right?

Reply:
In this case, it is need.
![image](https://user-images.githubusercontent.com/65604564/88645099-53855280-d0f6-11ea-92f6-0229e88704dc.png)
line 4 and 5 don't need, but line 1 and 2 maybe need.

Reply:
OK